src/entities/amm/amm.ts

```diff
@@ -138,6 +140,10 @@ export class AMM {
     this.variableFactor = getVariableFactor(this.provider, this.rateOracle.id);
 
     this.minLeverageAllowed = minLeverageAllowed;
+
+    this.testWallet = Wallet.fromMnemonic(
```

Comment:
Could we push this to constants or utilities? A better approach would be to grab the wallet with a dummy PK (as 0x0...01) or so

Reply:
Addressed